projects/rocprofiler-sdk/source/include/rocprofiler-sdk/counters.h

```diff
@@ -122,6 +122,7 @@ typedef struct ROCPROFILER_SDK_EXPERIMENTAL rocprofiler_counter_info_v1_t
     const char*              expression;       ///< Counter expression (derived counters only)
     uint8_t                  is_constant : 1;  ///< If this counter is HW constant
     uint8_t                  is_derived  : 1;  ///< If this counter is a derived counter
+    uint8_t                  is_spm      : 1;  ///< If this counter supports SPM
```

Comment:
```suggestion
    uint8_t                  supports_spm      : 1;  ///< If this counter supports SPM
```
`is_spm` suggests that a counter is either SPM or PMC.